style.css

```diff
@@ -0,0 +1,167 @@
+/* removes text under-line */
+A {
+  text-decoration: none;
+}
+
+
+.nav-bar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    min-height: 8vh;
+}
+
+.atube-left {
+    display: flex;
+    width: 20%;
+    
+}
+#logo {
+    color: red; 
+}
+
+
+.fas {
+  padding-right: 20px;
+  padding-left: 10px;
+}
+
+a {
+  color: black;
+  
+}
+
+.atube-left {
+  width: 20%;
+  display: flex;
+}
+
+.atube-right {
+  display: flex;
+  width: 20%;
+  justify-content: flex-end;
+  align-items: flex-end;
+}
+
+
+
+/* source code-stackoverflow, now modified */
+input[type="text"] {
+  width: 600px;
+  height: 26px;
+  padding-right: 50px;
+  justify-content: center;
+  margin-right: 70px;
+}
+
+button[type="search-icons"] {
+  margin-left: -74.5px;
+  height: 31.2px;
+  width:75px;
+
+}
+
+
+.container-left {
+  
+  width: 65%;
+  height: 45em;
+  float: left;
+  padding-left: 30px;
+  
+}
+
+.video {
+  width: 980px;
+  height: 540px;
+}
+
+.container-right {
+  
+  width: 30%;
+  height: 45em;
+  float: right;
+}
+
+
+
+.os-contibution {
+  display: flex;
+  justify-content: space-between;
+  
+}
+
+.subscribers {
+  display: flex;
+ 
+  justify-content: space-between;
+  align-items: center;
+  
+}
+.views {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  
+  width: 60%;
+ 
+}
+
+ .button {
+  background-color: red;
+  color: white;
+  font-size: 16px;
+  text-align: center;
+}
+
+.likes {
+  display: flex;
+  justify-content: space-around;
+  width: 40%;
+  align-items: center;
+  
+}
+/* removes list-item marker*/
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  
+}
+
+.video-img-1 {
+  display: flex;
+  align-items: center;
+}
+
+.video-img-2 {
+  display: flex;
+  align-items: center;
+}
+
+
+.video-img-3 {
+  display: flex;
+  align-items: center;
+}
+
+.video-img-4 {
+  display: flex;
+  align-items: center;
+}
+
+.video-img-5 {
+  display: flex;
+  align-items: center;
+}
+
+.video-img-6 {
+  display: flex;
+  align-items: center;
+}
+
+.video-img-7 {
+  display: flex;
+  align-items: center;
+}
+
```

Comment:
All of these elements have the same properties. You can actually select them all and apply this rule only once!